redundant-bmc/src/manager.cpp

```diff
@@ -17,9 +21,11 @@ Manager::Manager(sdbusplus::async::context& ctx) :
 // NOLINTNEXTLINE
 sdbusplus::async::task<> Manager::startup()
 {
-    // TODO: Actually figure out the role
-    lg2::info("Setting role to passive");
-    redundancyInterface.role(RedundancyInterface::Role::Passive);
+    ctx.spawn(doHeartBeat());
+
+    auto role = determineRole();
```

Comment:
We could remove `role` temporary variable and call `determineRole` in the below line.

Reply:
I have changes coming in this area in the future, but yea that would be better for now.